tests/core/data/test_data_manager.py

```diff
@@ -45,6 +45,24 @@ def test_create_data_node_and_modify_properties_does_not_modify_config(self):
         assert dn.properties.get("foo") == "bar"
         assert dn.properties.get("baz") == "qux"
 
+    def test_can_create(self):
+        dn_config = Config.configure_in_memory_data_node("dn", 10, scope=Scope.SCENARIO)
+        global_dn_config = Config.configure_data_node(
+            id="global_dn", storage_type="in_memory", scope=Scope.GLOBAL, data=10
+        )
+
+        reasons = _DataManager._can_create(global_dn_config)
+        assert bool(reasons) is True
```

Comment:
Do we really need to cast the Reasons object? What would the following do?
```
assert reasons is True
```
Does it work as well?

Reply:
Unfortunately, no. It's a Reasons object

Reply:
and what about?
```
assert reasons==True
```

Reply:
unfortunately, it won't work

Reply:
I thought the Reasons object would behave like a boolean in every situation. 
That is not so important. The most important is to be able to do: 
`if reasons:`

Reply:
yes we can do that, since the `if reasons:` is the same as `if bool(reasons):`